pyproject.toml

```diff
@@ -12,6 +12,8 @@ dependencies = [
   "aiohttp>=3",
   "tzdata>=2024.2 ; platform_system == 'Windows'",
   "mashumaro>=3.14",
+  "sphinx-autodoc-typehints[docs]>=1.23.0",
+  "sphinx[docs]>=7.4.7",
```

Comment:
These should not be in our runtime dependencies as they are only used for generating docs.

Reply:
Aii, good spot. I did `uv add --extra docs` instead of `uv add --optional docs`